PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+  - no variant type indicated for all products that are neither a master product nor a variant
+
+
+- *Attribute Set*   
+  Assigned attribute set.
+
+- *ID*   
+  Product identification number. The ID number is automatically assigned by the system when the product is created.
+
+- *Modified on*   
+  Date and time of the last modification.
+
+- *Modified by*   
+  Name and username of the user who modified the product.
+
+- *Created on*   
+  Date and time of the creation.
+
+- *Created by*   
+  Name and username of the user who created the product.
+
+- Attribute    
+  You can add a column for each attribute that is assigned to the product. The column displays the attribute name, the row displays the corresponding attribute value of the product.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click this button to create a product. The *Create new product* window is displayed.   
+  For detailed information, see [Create a product](../Operation/01_ManageProducts.md#create-a-product).
+
+
+## Create view
+*PIM > Products > Tab LIST > Button Points > Menu entry create*
+
+![Create view](../../Assets/Screenshots/PIM/Products/List/CreateView.png "[Create view]")
+
+- *Name*   
+  Enter a name for the view.
+
+- [CANCEL]   
+  Click this button to cancel creating a view. The *Create view* window is closed.
+
+- [SAVE]   
+  Click this button to save the new view. The *Create view* window is closed. The view is saved and displayed in the *View* drop-down list.
+
+
+## Rename view
+*PIM > Products > Tab LIST > Button Points > Menu entry rename*
+
+![Rename view](../../Assets/Screenshots/PIM/Products/List/RenameView.png "[Rename view]")
+
+- *Name*   
+  Click this field to edit the views name.
+
+- [CANCEL]   
+  Click this button to cancel renaming the view. The *Rename view* window is closed.
+
+- [SAVE]   
+  Click this button to save the changes to the views name. The *Rename view* window is closed. The new name for the view is saved and displayed in the *View* drop-down list.
+
+
+## Create new product
+*PIM > Products > Tab LIST > Button Add*
+
+![Create new product](../../Assets/Screenshots/PIM/Products/List/CreateNewProduct.png "[Create new product]")
+
+- *Attribute set*   
+  Click the drop-down list to select the attribute set for the new product. All available attribute sets are displayed in the drop-down list.
+
+- *SKU*   
+  Enter the SKU of the new product. The SKU (Stock Keeping Unit) is an identification number for the product. Therefore, it should be uniquely assigned to a single product.
+
+
+## Create product
+*PIM > Products > Tab LIST > Button Add > Button CREATE*
+
+![Create Product](../../Assets/Screenshots/PIM/Products/List/CreateProduct.png "[Create Product]")
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+  Click this button to close the *Create product* view and return to the product list. All changes are rejected.
+
+- [CANCEL]   
+  Click this button to cancel the creation of a product.
```

Comment:
```suggestion
  Click this button to cancel creating of a product.
```
Bei Create view -> cancel the creation vs. creating -> vereinheitlichen und in Standardformulierungen